check_version.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+#
+# Copyright Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+CACHED_CSV_VERSION=${CACHED_CSV_VERSION:-'.cache/csv_version.txt'}
+CACHED_CSV_CONTAINER_IMAGE_TAG=${CACHED_CSV_CONTAINER_IMAGE_TAG:-'.cache/csv_container_image_tag.txt'}
+CACHED_CSV_NAME_TAG=${CACHED_CSV_NAME_TAG:-'.cache/csv_name_tag.txt'}
+CACHED_BUNDLE_VERSION=${CACHED_BUNDLE_VERSION:-'.cache/bundle_version.txt'} 
+CACHED_BUNDLE_CONTAINER_IMAGE_TAG=${CACHED_BUNDLE_CONTAINER_IMAGE_TAG:-'.cache/bundle_container_image_tag.txt'}
+CACHED_BUNDLE_NAME_TAG=${CACHED_BUNDLE_NAME_TAG:-'.cache/bundle_name_tag.txt'}
+CACHED_MANAGER_IMAGE_TAG=${CACHED_MANAGER_IMAGE_TAG:-'.cache/manager_image_tag.txt'}
+
+ref_name=$1
+is_ci=${2:-"false"}
```

Comment:
CI platforms usually inject default environment variables to detect if we are running in CI or not. For instance, GitHub always sets a `CI` env var (set to `true`): https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
Would it make sense to use that instead?

Reply:
resolved in https://github.com/devfile/registry-operator/pull/56/commits/dab9fda11a1572aef75eacddd64b0e8a5600cb3c